eng/native/configurecompiler.cmake

```diff
@@ -219,6 +219,9 @@ elseif (CLR_CMAKE_HOST_ARCH_ARM64)
 elseif (CLR_CMAKE_HOST_ARCH_S390X)
   set(ARCH_HOST_NAME s390x)
   add_definitions(-DHOST_S390X -DHOST_64BIT -DBIGENDIAN)
+elseif (CLR_CMAKE_HOST_ARCH_LOONGARCH64)
```

Comment:
Nit: Move it up right after CLR_CMAKE_HOST_ARCH_ARM64

Reply:
OK, Thanks